returnn/tensor/_dim_extra.py

```diff
@@ -1184,6 +1184,8 @@ def _bin_op_tf(a, b):
             elif kind in ("floordiv", "truediv"):  # truediv assumes there is no remainder
                 return a // b
```

Comment:
For completeness, should I also add it here?

Reply:
Yes.